tests/test_rate_limiter.py

```diff
@@ -63,7 +63,7 @@ def test_rate_limiter_with_redis(limiter_test_normal, mock_wikipedia_response):
     Test that Idunn stops external requests when
     we are above the max rate
 
-    We mock 5 calls to wikipedia while the max number
+    We mock 5*2 calls to wikipedia while the max number
     of calls is 3*2, so we test that after 3 calls
     no more calls are done
     """
```

Comment:
This comment could probably be made clearer.  

5 requests to Idunn, and each of them requires 2 blocks ("description" + "wikipedia") * 2 requests ("title" + "summary") = 4 calls per POI request.  
The limit is set to 12 calls to Wikipedia, so after 3 POI requests, these blocks are no longer present.